inc/cleantalk-common.php

```diff
@@ -593,6 +594,29 @@ function apbct_js_keys__get__ajax()
     die(json_encode(array('js_key' => ct_get_checkjs_value())));
 }
 
+/**
+ * Checking email before POST
+ */
+function apbct_email_check_before_post()
+{
+    global $apbct;
+    $ajax_type = $apbct->data['ajax_type'];
+    $email = isset($_POST['data']['email']) ? trim($_POST['data']['email']) : null;
+
+    if ($ajax_type === 'rest') {
+        $email = trim(Post::get('email'));
+    }
```

Comment:
Code above can be replaced to `$email = trim(Post::get('email'));`

Reply:
Done